internal/sql/repository/team/TeamRepository.go

```diff
@@ -92,15 +92,15 @@ func (repo TeamRepositoryImpl) FindTeamByAppName(appName string) (*Team, error)
 	team := &Team{}
 	err := repo.dbConnection.Model(team).Column("team.*").
 		Join("inner join app a on a.team_id = team.id").Where("a.app_name = ?", appName).
-		Where("a.active = ?", true).Select()
+		Where("team.active = ?", true).Where("a.active = ?", true).Select()
 	return team, err
 }
 
 func (repo TeamRepositoryImpl) FindTeamByAppNameV2() ([]*TeamRbacObjects, error) {
```

Comment:
Fix the name, it is not getting team but rbacObjects also only active

Reply:
removed, not in used